CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe Client Python SDK will be documented in this file.
 
+## Recent Changes
+
+- Add method to issue SSH commands. [#253](https://github.com/zowe/zowe-client-python-sdk/issues/253)
```

Comment:
This changelog entry should be in past tense. It would also be good to include the method that was added: "Added `bbbb` method to issue SSH commands"